akka-docs-new/src/test/scala/tutorial_3/IotApp.scala

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package tutorial_3
+
+import akka.actor.ActorSystem
+import tutorial_3.DeviceManager.RequestTrackDevice
+
+import scala.io.StdIn
+
+object IotApp {
+
+  def main(args: Array[String]): Unit = {
+    val system = ActorSystem("iot-system")
+
+    try {
+      // Create top level supervisor
+      val supervisor = system.actorOf(DeviceManager.props(), "iot-supervisor")
+
+      supervisor ! RequestTrackDevice("mygroup", "device1")
+
+      // Exit the system after ENTER is pressed
+      StdIn.readLine()
+    } finally {
+      system.terminate()
+    }
```

Comment:
Since this is for complete beginners I don't want to give them the impression that everything here can blow up (code in the try currently really has nothing scary, won't throw)